tests/visual/test_show_map.py

```diff
@@ -126,3 +126,15 @@ def test_show_map_crs_change_to_4326_2(layer_polygon, layer_points, layer_polygo
     QgsProject.instance().addMapLayers(
         [layer_points, layer_polygon_3067, layer_polygon]
     )
+
+
+@pytest.mark.qgis_show_map(timeout=DEFAULT_TIMEOUT, zoom_to_common_extent=False)
+def test_show_map_should_not_keep_the_set_extent(
```

Comment:
Could you please clarify the function name. I don't understand what this should test by the name. Keep what extent, keep how?

Reply:
Yeah, it was a bit strange name...